LICENSE

```diff
@@ -1,6 +1,10 @@
 MIT License
 
-Copyright (c) 2021 David Widmann
+Copyright (c) 2021 Brian J Smith, David Widmann and contributors:
+
```

Comment:
```suggestion
https://turing.ml/dev/team/
```

Reply:
@yebai, I've updated it. Was a bit confused by your suggestion though, so I hope that I did it right

Reply:
looks great - many thanks @rikhuijzer!